src/components/HRInfoAddress.js

```diff
@@ -0,0 +1,154 @@
+import React          from 'react';
```

Comment:
@AlexisReliefApps Why did you add this file instead of modifying the existing Address.js file ?